packages/kotti-ui/source/kotti-style/_comments.scss

```diff
@@ -1,127 +1,128 @@
-.comment {
+.kt-comment {
```

Comment:
can u move this entire file in the scope of `KtComment` 
this way of defining the styles is no longer how we'd do it IMO

Reply:
we had `kotti-styles` folder when the css styling used to be a separate package from kotti, but this has long since changed, and so we can already inline styles or if the styling is huge, we define it on a styles file under the folder of the component